api/app/signals/apps/questionnaires/tests/test_utils_mermaidx.py

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Gemeente Amsterdam
+from unittest import skip
+
+from django.test import TestCase
+
+from signals.apps.questionnaires.services import QuestionGraphService
+from signals.apps.questionnaires.tests.data.questionnaires import questionnaire_geluidsoverlast
+from signals.apps.questionnaires.utils.mermaidx import (
+    DIRECTION_DT,
+    DIRECTION_LR,
+    DIRECTION_RL,
+    DIRECTION_TD,
+    mermaidx
+)
+
+
+class TestUtilMermaid(TestCase):
+    @skip('When running this test, the questions in the graph do not always have the expected id\'s')
```

Comment:
This can be worked around by using `question.ref` in stead of `question.id`.

Reply:
Fixed